### PR TITLE
init

### PR DESCRIPTION
**Pull request by SDT-Ilya-sh**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
     public static void main(String[] args)
     {
+        int a;
         double incomeNeedForInvestments = 0.1;
         while (true) {
             System.out.println("Введите сумму доходов компании за месяц " +
```
Comment (zhenekq) [inline src/Main.java]: ну норм,  но давай по новой

Comment (SDT-Ilya-sh) [reply src/Main.java]: ну ты и душнила

**Review by zhenekq — COMMENTED** [review]
nice

File: src/Main.java (hunk 1 above)
Comment (zhenekq) [inline src/Main.java]: dafgsdfg

File: src/Main.java [hunk 2]
```diff
@@ -18,6 +18,8 @@
 
     public static void main(String[] args)
     {
+        int gitCheck;
+        boolean gitProverka;
         double incomeNeedForInvestments = 0.1;
         while (true) {
             System.out.println("Введите сумму доходов компании за месяц " +
```
Comment (zhenekq) [inline src/Main.java]: харош, норм написал, ебашишь бро

**Review by zhenekq — COMMENTED** [review]
Take out logic in separate class, entity must have only contructors, getters, setters,  not logic of entity

File: src/Main.java (hunk 2 above)
Comment (SDT-Ilya-sh) [reply src/Main.java]: outdated but thanks

File: src/lesson1/Cat.java [hunk 3]
```diff
@@ -41,6 +48,7 @@ public Double getWeight()
     public String getStatus()
```
Comment (zhenekq) [inline src/lesson1/Cat.java:48]: Take out logic in separate class, entities must have only construnctors, getters, setter

File: src/lesson1/Cat.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+    public void feed(Double amount)
+    {
+        weight = weight + amount;
+    }
+
+    public void drink(Double amount)
+    {
+        weight = weight + amount;
+    }
```
Comment (zhenekq) [inline src/lesson1/Cat.java]: Take out logic in separate class, entities must have only construnctors, getters, setter

File: src/lesson1/Cat.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            return "Exploded";
+        }
+        else if(weight > originWeight) {
+            return "Sleeping";
+        }
+        else {
+            return "Playing";
+        }
+    }
+}
```
Comment (zhenekq) [inline src/lesson1/Cat.java]: Also take out logic in separate class

File: src/lesson1/Loader.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package lesson1;
+
+public class Loader
+{
+    public static void main(String[] args)
+    {
+        Cat cat = new Cat();
+        System.out.println("Proverka Zheki");
+        System.out.println(cat.getStatus());
+    }
```
Comment (zhenekq) [inline src/lesson1/Loader.java]: Create 4 cats, show console result at comment in java code, like this // 4 cats created, 1 deleted

**Review by SDT-Ilya-sh — COMMENTED** [review]
Не палуччаецца

**Review by SDT-Ilya-sh — COMMENTED** [review]
Не вижу Жеку

File: src/lesson1/Colors.java [hunk 7]
```diff
@@ -0,0 +1,9 @@
+package lesson1;
+
+public enum Colors {
+    BLACK,
+    WHITE,
+    BLACK_AND_WHITE,
+    RED,
+    TRICOLOR,
```
Comment (zhenekq) [inline src/lesson1/Colors.java:8]: I think main theme of your task include this Enum to your class Cat like a field and try to manupalaite with it

Comment (SDT-Ilya-sh) [reply src/lesson1/Colors.java:8]: no its not(yet)

File: src/lesson2/Camera.java [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+        String randomNumber = Double.toString(Math.random()).substring(2, 5);
+        int randomHeight = (int) (1000 + 3500. * Math.random());
+        double randomWeight = 600 + 10000 * Math.random();
+
+        Car car = new Car();
+        car.number = randomNumber;
+        car.height = randomHeight;
+        car.weight = randomWeight;
+        car.hasVehicle = Math.random() > 0.5;
+        car.isSpecial = Math.random() < 0.15;
```
Comment (zhenekq) [inline src/lesson2/Camera.java]: make with using getters and setters

File: src/lesson2/Car.java [hunk 9]
```diff
@@ -0,0 +1,61 @@
+package lesson2;
+
+public class Car {
+    public static String number;
+    public static int height;
+    public static double weight;
+    public static boolean hasVehicle;
+    public static boolean isSpecial;
```
Comment (zhenekq) [inline src/lesson2/Car.java]: make private

Comment (zhenekq) [reply src/lesson2/Car.java]: delete static

File: src/lesson2/Camera.java [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+        }
+         public static Double weightCheck()
+        {
+
+            getNextCar().setWeight(1488.00);
+            return Car.getWeight();
+
+
+        }
```
Comment (zhenekq) [inline src/lesson2/Camera.java]: UBRATb NAHUY

File: src/lesson2/Camera.java (hunk 8 above)
Comment (zhenekq) [reply src/lesson2/Camera.java]: print values of object with using getters

File: src/lesson1/Cat.java [hunk 11]
```diff
@@ -11,6 +11,7 @@
     private double weight;
     private double minWeight;
     private double maxWeight;
+    public String catColor;
```
Comment (zhenekq) [inline src/lesson1/Cat.java]: write with using Enum Color(Black, Brown, White)

File: src/lesson1/Loader.java [hunk 12]
```diff
@@ -12,6 +12,9 @@ public static void main(String[] args) {
 
         getKitten();
         getKitten();
+        Cat white = new Cat("White");
+        System.out.println(white.getCatColor());
+
```
Comment (zhenekq) [inline src/lesson1/Loader.java]: create three cats

File: src/lesson1/Cat.java (hunk 4 above)
Comment (SDT-Ilya-sh) [reply src/lesson1/Cat.java]: fixed

File: src/lesson1/Cat.java (hunk 5 above)
Comment (SDT-Ilya-sh) [reply src/lesson1/Cat.java]: fixed

File: src/lesson1/Loader.java (hunk 6 above)
Comment (SDT-Ilya-sh) [reply src/lesson1/Loader.java]: done

File: src/lesson2/Car.java (hunk 9 above)
Comment (SDT-Ilya-sh) [reply src/lesson2/Car.java]: fixed

File: src/lesson2/Camera.java (hunk 8 above)
Comment (SDT-Ilya-sh) [reply src/lesson2/Camera.java]: fixed

File: src/lesson2/Camera.java (hunk 10 above)
Comment (SDT-Ilya-sh) [reply src/lesson2/Camera.java]: UBRAL NAHUY!

File: src/lesson1/Cat.java (hunk 11 above)
Comment (SDT-Ilya-sh) [reply src/lesson1/Cat.java]: done

File: src/lesson1/Loader.java (hunk 12 above)
Comment (SDT-Ilya-sh) [reply src/lesson1/Loader.java]: created

File: src/lesson1/CatLogic.java [hunk 13]
```diff
@@ -0,0 +1,55 @@
+package lesson1;
+
+public class CatLogic
+{
+    public static int count=0;
+    private double originWeight;
+    private double weight;
+    private double minWeight;
+    private double maxWeight;
+    public String catColor;
```
Comment (zhenekq) [inline src/lesson1/CatLogic.java:10]: logic of entitry with fields of entity, u should accept in methods, entity

File: src/lesson1/CatLogic.java [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+            return "Exploded";
+        }
+        else if(weight > originWeight) {
+            return "Sleeping";
+        }
+        else {
+            return "Playing";
+        }
+    }
+}
```
Comment (zhenekq) [inline src/lesson1/CatLogic.java:55]: Send in these methods smth like this public void meow(Cat cat){logic...} (delete fields of entity in this class)

**Review by zhenekq — COMMENTED** [review]
very nice, good job

**Review by zhenekq — COMMENTED** [review]
also good job

File: src/lesson1/CatLogic.java (hunk 13 above)
Comment (zhenekq) [inline src/lesson1/CatLogic.java]: fields of entity only in entity class, you should accept Cat in logic methods

File: src/lesson1/CatLogic.java (hunk 14 above)
Comment (zhenekq) [inline src/lesson1/CatLogic.java]: make smth like this, accept Cat in methods public void meow(Cat cat){logic...}

**Review by zhenekq — COMMENTED** [review]
nice bro

**Review by zhenekq — COMMENTED** [review]
good job

File: src/lesson5/Main.java [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        int sum = 0;
+        String numberToString = number.toString();
+        for (int i=0;i < numberToString.length(); i++){
+            String charNumber = String.valueOf(numberToString.charAt(i));
+            int stringNumber = Integer.parseInt(charNumber);
+            sum+=stringNumber;
+        }
+        return sum;
+    }
```
Comment (zhenekq) [inline src/lesson5/Main.java]: ты научишься выносить логику в отдельный класс? Чувак ты серьезно...

Comment (SDT-Ilya-sh) [reply src/lesson5/Main.java]: fixed honey

File: src/lesson7/Formation.java [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+
+                contNumber++;
+                contNumberInTruck++;
+
+            }
+
+             boxNumberInContainer = 1;
+             contNumberInTruck = 1;
+
+        }
```
Comment (zhenekq) [inline src/lesson7/Formation.java]: devide this on more little tasks, not a one huge, it will be useful for u

**Review by zhenekq — COMMENTED** [review]
Better, nice

**Review by SDT-Ilya-sh — COMMENTED** [review]
https://www.youtube.com/watch?v=RgN3VIfB59o

File: src/lesson8/Main.java [hunk 17]
```diff
@@ -0,0 +1,8 @@
+package lesson8;
+
+public class Main {
+    public static void main(String[] args) {
+    NewString string = new NewString();
+    string.codePrinting("Жека читает мой код");
```
Comment (zhenekq) [inline src/lesson8/Main.java]: you dont need object of this class, make util methods static!

Comment (zhenekq) [reply src/lesson8/Main.java]: you can involve ulit methods without creating object of this class.
for example: =NewString.codePrinting(....); and u will have the same result

Comment (zhenekq) [reply src/lesson8/Main.java]: with using static sure

File: src/lesson9/Loader.java [hunk 18]
```diff
@@ -0,0 +1,12 @@
+package lesson9;
+
+public class Loader
+{
+    public static void main(String[] args)
+    {
+        SalaryCount newTry = new SalaryCount();
+        newTry.setSalary("Вася наебланил 322 рублей, Петя насосал - 1337 рубля, а Маша на заводе - 228 рублей");
```
Comment (zhenekq) [inline src/lesson9/Loader.java]: the same as the previous

Comment (SDT-Ilya-sh) [reply src/lesson9/Loader.java]: fixed

File: src/lesson8/Main.java (hunk 17 above)
Comment (SDT-Ilya-sh) [reply src/lesson8/Main.java]: fixed

**Pull request merged**